provider/clusterrosaclassic/cluster_rosa_classic_resource.go

```diff
@@ -1053,6 +1018,42 @@ func (r *ClusterRosaClassicResource) Read(ctx context.Context, request resource.
 	response.Diagnostics.Append(diags...)
 }
 
+func assertNoChanges(state, plan *ClusterRosaClassicState) diag.Diagnostics {
+	diags := diag.Diagnostics{}
+	common.AssertStateAndPlanValue(state.Name, plan.Name, "name", &diags)
+	common.AssertStateAndPlanValue(state.ExternalID, plan.ExternalID, "external_id", &diags)
+	common.AssertStateAndPlanValue(state.MultiAZ, plan.MultiAZ, "multi_az", &diags)
+	common.AssertStateAndPlanValue(state.DisableSCPChecks, plan.DisableSCPChecks, "disable_scp_checks", &diags)
+	common.AssertStateAndPlanValue(state.Tags, plan.Tags, "tags", &diags)
+	common.AssertStateAndPlanValue(state.EtcdEncryption, plan.EtcdEncryption, "etcd_encryption", &diags)
+	common.AssertStateAndPlanValue(state.BaseDNSDomain, plan.BaseDNSDomain, "base_dns_domain", &diags)
+	common.AssertStateAndPlanValue(state.AWSAccountID, plan.AWSAccountID, "aws_account_id", &diags)
+	common.AssertStateAndPlanValue(state.AWSSubnetIDs, plan.AWSSubnetIDs, "aws_subnet_ids", &diags)
+	common.AssertStateAndPlanValue(state.KMSKeyArn, plan.KMSKeyArn, "kms_key_arn", &diags)
+	common.AssertStateAndPlanValue(state.FIPS, plan.FIPS, "fips", &diags)
+	common.AssertStateAndPlanValue(state.AWSPrivateLink, plan.AWSPrivateLink, "aws_private_link", &diags)
+	common.AssertStateAndPlanValue(state.Private, plan.Private, "private", &diags)
+	common.AssertStateAndPlanValue(state.AvailabilityZones, plan.AvailabilityZones, "availability_zones", &diags)
+	common.AssertStateAndPlanValue(state.MachineCIDR, plan.MachineCIDR, "machine_cidr", &diags)
+	common.AssertStateAndPlanValue(state.ServiceCIDR, plan.ServiceCIDR, "service_cidr", &diags)
+	common.AssertStateAndPlanValue(state.PodCIDR, plan.PodCIDR, "pod_cidr", &diags)
+	common.AssertStateAndPlanValue(state.HostPrefix, plan.HostPrefix, "host_prefix", &diags)
+	common.AssertStateAndPlanValue(state.ChannelGroup, plan.ChannelGroup, "channel_group", &diags)
+	common.AssertStateAndPlanValue(state.Ec2MetadataHttpTokens, plan.Ec2MetadataHttpTokens, "ec2_metadata_http_tokens", &diags)
+	common.AssertStateAndPlanValue(state.AWSAdditionalControlPlaneSecurityGroupIds, plan.AWSAdditionalControlPlaneSecurityGroupIds, "aws_additional_control_plane_security_group_ids", &diags)
+	common.AssertStateAndPlanValue(state.AWSAdditionalInfraSecurityGroupIds, plan.AWSAdditionalInfraSecurityGroupIds, "aws_additional_infra_security_group_ids", &diags)
```

Comment:
Missing `aws_additional_control_plane_security_group_ids`

Reply:
thank you! nice catch